[Build Error] Gradle Error

Hello, 

I have installed all the dependencies that should make building Kaiju extension possible according to the guidelines that are mentioned [here ](https://github.com/CERTCC/kaiju/blob/main/INSTALL.md), however I am getting the following error regading the documentation build. 

> PS C:\Users\workspace\kaiju> gradle 
> 
> > Configure project :
> Using Ghidra install directory C:/Users/nxf76051/workspace/ghidra_11.0_PUBLIC_20231222/ghidra_11.0_PUBLIC
> Using Java 17 for Ghidra 11.0
> Using AUTOCATS test directory null
> Targeting operating system 'Windows 11' (windows 11-amd64)
> We are adding a dependency
> 
> > Task :markdownToGhidraHtml FAILED
> 
> > Task :downloadZ3Source
> Download https://github.com/Z3Prover/z3/archive/refs/tags/z3-4.12.1.zip
> 
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> A problem was found with the configuration of task ':markdownToGhidraHtml' (type 'MarkdownToGhidraHtmlTask').
>   - Gradle detected a problem with the following location: 'C:\Users\workspace\kaiju\src\main\help\help\topics\kaiju'.
> 
>     Reason: Task ':processResources' uses this output of task ':markdownToGhidraHtml' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
> 
>     Possible solutions:
>       1. Declare task ':markdownToGhidraHtml' as an input of ':processResources'.
>       2. Declare an explicit dependency on ':markdownToGhidraHtml' from ':processResources' using Task#dependsOn.
>       3. Declare an explicit dependency on ':markdownToGhidraHtml' from ':processResources' using Task#mustRunAfter.
> 
>     Please refer to https://docs.gradle.org/8.0.1/userguide/validation_problems.html#implicit_dependency for more details about this problem.
> 
> * Try:
> > Run with --stacktrace option to get the stack trace.
> > Run with --info or --debug option to get more log output.
> > Run with --scan to get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> BUILD FAILED in 4s
> 7 actionable tasks: 2 executed, 5 up-to-date

Any idea what could have gone wrong?

--- a/build.gradle
+++ b/build.gradle
@@ -361,6 +361,7 @@ indexHelp.dependsOn(copyToLib)
 indexHelp.dependsOn(markdownToGhidraHtml)
 buildHelp.dependsOn(markdownToGhidraHtml)
 zipSource.dependsOn(markdownToGhidraHtml)
+processResources.dependsOn(markdownToGhidraHtml)
 markdownToGhidraHtml.sourceDir = file('docs/')
 markdownToGhidraHtml.outputDir = file('src/main/help/help/topics/kaiju/')
 

